doc/Building.md

```diff
@@ -22,8 +22,8 @@ Install the prerequisites using APT:
 
 ```
 sudo apt-get install build-essential git patch wget unzip \
-gettext autoconf automake cmake libtool nasm ragel luarocks lua5.1 libsdl2-dev \
-libssl-dev libffi-dev libsdl2-dev libc6-dev-i386 xutils-dev linux-libc-dev:i386 zlib1g:i386
```

Comment:
I was more tired yesterday than I thought. I somehow read sdl2 as sdl12 xD

Reply:
Er, though I just realized it's in there twice so removing it was the correct call by accident. :facepalm: